src/index.js

```diff
@@ -136,7 +138,11 @@ if (window.CanvasRenderingContext2D) {
       const tool = docViewer.getTool(window.Tools.ToolNames.STICKY);
       tool?.setSaveViewState(true);
     }
-
+    docViewer.on('documentLoaded', () => {
```

Comment:
Let's match the change that is made in master

Reply:
The code has been updated with the other branch